frontend/e2e/annotation-mode-page/annotation-mode.page.spec.ts

```diff
@@ -221,6 +244,62 @@ test.describe("Annotate page", () => {
 });
 
 test.describe("Annotation page shortcuts", () => {
+  test.describe("Global shortcuts", () => {
+    test("when focus is not in form, press Tab to focus on first question", async ({ page }) => {
+      await goToAnnotationPage(page);
+      const formBox = await page.getByText("Submit your feedback").boundingBox();
+
+      await page.mouse.click(formBox.x - 25, formBox.y - 25); // click outside form to loose focus
+      await expect(page).toHaveScreenshot();
+
+      await page.keyboard.press(shortcuts.focusOnForm);
+      await expect(page).toHaveScreenshot();
+    });
+    test("on first record, go to next record by pressing Ctrl+ArrowRight", async ({ page }) => {
```

Comment:
This test is related to pagination? could you please put inside a describe related on that? just to find it more easily